cluster/examples/kubernetes/ceph/common.yaml

```diff
@@ -231,6 +235,8 @@ spec:
                   replicated:
                     properties:
                       size:
+                        maximum: 10
+                        minimum: 0
```

Comment:
```suggestion
                        minimum: 1
```
?

Reply:
the minimum on pool sizes actually needs to be 0 because you have to either set the replicated.size or the erasure coded sizes, but not both.

Reply:
The Ceph Pool's size property is ignored, if it is an erasure coded pool